diff --git a/install.sh b/install.sh
index 4d00714..23cd925 100755
--- a/install.sh
+++ b/install.sh
@@ -609,33 +609,33 @@ else
       exit 1
   fi
 
-  #Setting openGL driver and GPU memory to 128mb
-  if $isRpi; then
-    sudo raspi-config nonint do_memory_split 128
-    if [[ $? -eq 0 ]]; then
-      echo -e Memory set to 128mb'\n'
-    else
-      echo Setting memory failed with error code $? please set manually
-      exit 1
-    fi
-
-    sudo raspi-config nonint do_gldriver G2
-    if [[ $? -eq 0 ]]; then
-      echo -e OpenGL set ok'\n'
-    else
-      echo Setting openGL failed with error code $? please set manually
-      exit 1
-    fi
-
-    echo enabling krnbt to speed up boot and improve stability
-    cat <<EOT >> /boot/config.txt
-      dtparam=krnbt
-EOT
-  fi
-
-
-  #Start app
-  echo Starting app
-  cd ../bin
-  ./dash
+#  #Setting openGL driver and GPU memory to 128mb
+#  if $isRpi; then
+#    sudo raspi-config nonint do_memory_split 128
+#    if [[ $? -eq 0 ]]; then
+#      echo -e Memory set to 128mb'\n'
+#    else
+#      echo Setting memory failed with error code $? please set manually
+#      exit 1
+#    fi
+#
+#    sudo raspi-config nonint do_gldriver G2
+#    if [[ $? -eq 0 ]]; then
+#      echo -e OpenGL set ok'\n'
+#    else
+#      echo Setting openGL failed with error code $? please set manually
+#      exit 1
+#    fi
+#
+#    echo enabling krnbt to speed up boot and improve stability
+#    cat <<EOT >> /boot/config.txt
+#      dtparam=krnbt
+#EOT
+#  fi
+#
+#
+#  #Start app
+#  echo Starting app
+#  cd ../bin
+#  ./dash
 fi
