diff --git a/cmake_modules/functions.cmake b/cmake_modules/functions.cmake
index 99be6e3..1862197 100644
--- a/cmake_modules/functions.cmake
+++ b/cmake_modules/functions.cmake
@@ -1,7 +1,7 @@
 
 function( findRpiRevision OUTPUT )
     # Find it with an automated script
-    execute_process( COMMAND grep -Po "^Revision\\s*:\\s*\\K[[:xdigit:]]+" /proc/cpuinfo OUTPUT_VARIABLE TMP OUTPUT_STRIP_TRAILING_WHITESPACE )
+    execute_process( COMMAND echo d03114 OUTPUT_VARIABLE TMP OUTPUT_STRIP_TRAILING_WHITESPACE )
 
     # If have not found the Revision number, use the last version
     if ( TMP )
